src/components/WwwFrame/EndOfYearAppealBanner/AppealBanner.vue

```diff
@@ -101,19 +93,68 @@ export default {
 	data() {
 		return {
 			open: true,
+			appealEnabled: false,
+			appealMatchEnabled: false,
+			appealBonusEnabled: false,
+			isAppealShrunk: false,
 		};
 	},
 	apollo: {
 		query: appealBannerQuery,
 		preFetch: true,
 		result({ data }) {
-			this.isVisible = _get(data, 'general.setting.value') === 'true' || false;
+			this.appealEnabled = settingEnabled(
+				data,
+				'general.appeal_enabled.value',
+				'general.appeal_start_time.value',
+				'general.appeal_end_time.value'
+			);
+
+			this.appealMatchEnabled = settingEnabled(
+				data,
+				'general.appeal_match_enabled.value',
+				'general.appeal_match_start_time.value',
+				'general.appeal_match_end_time.value'
+			);
+
+			// This setting SHOULD be temporary and CANNOT reveal this appeal alone.
+			this.appealBonusEnabled = readBoolSetting(data, 'general.appeal_bonus_active.value');
+
+			// the IsAppealBannerShrunk session value returns either false or '"1"'
+			this.isAppealShrunk = readJSONSetting(data, 'general.appeal_banner_shrunk.data') === 1;
```

Comment:
Check session for appeal shrunk, then watch this value below to close the appeal on load if this is set to true.